certification/engine/engine.go

```diff
@@ -124,34 +68,13 @@ func queryNewChecks(checkName string) certification.Check {
 }
 
 // Register all checks
-var deprecatedRunAsNonRootCheck certification.Check = &shell.RunAsNonRootCheck{}
-var deprecatedUnderLayerMaxCheck certification.Check = &shell.UnderLayerMaxCheck{}
-var deprecatedHasRequiredLabelCheck certification.Check = &shell.HasRequiredLabelsCheck{}
-var deprecatedBasedOnUbiCheck certification.Check = &shell.BaseOnUBICheck{}
-var deprecatedHasLicenseCheck certification.Check = &shell.HasLicenseCheck{}
-var deprecatedHasUniqueTagCheck certification.Check = &shell.HasUniqueTagCheck{}
-var deprecatedValidateOperatorBundle certification.Check = &shell.ValidateOperatorBundleCheck{}
-var deprecatedScorecardBasicSpecCheck certification.Check = &shell.ScorecardBasicSpecCheck{}
-var deprecatedScorecardOlmSuiteCheck certification.Check = &shell.ScorecardOlmSuiteCheck{}
-var deprecatedHasNoProhibitedCheck certification.Check = &shell.HasNoProhibitedPackagesCheck{}
-var deprecatedHasNoProhibitedMountedCheck certification.Check = &shell.HasNoProhibitedPackagesMountedCheck{}
-var deprecatedOperatorPkgNameIsUniqueMountedCheck certification.Check = &shell.OperatorPkgNameIsUniqueMountedCheck{}
-var deprecatedOperatorPkgNameIsUniqueCheck certification.Check = &shell.OperatorPkgNameIsUniqueCheck{}
-var hasMinimalVulnerabilitiesUnshareCheck certification.Check = &shell.HasMinimalVulnerabilitiesUnshareCheck{}
-var deprecatedDeployableByOlmCheck certification.Check = &k8s.DeployableByOlmCheck{}
-var deprecatedDeployableByOlmMountedCheck certification.Check = &k8s.DeployableByOlmMountedCheck{}
-
-// Disabled due to issue #99 and discussions in community meeting
-// var hasMinimalVulnerabilitiesCheck certification.Check = &shell.HasMinimalVulnerabilitiesCheck{}
-
-// new checks for CraneEngine
-var hasLicenseCheck certification.Check = &containerpol.HasLicenseCheck{}
-var hasUniqueTagCheck certification.Check = containerpol.NewHasUniqueTagCheck(internal.NewSkopeoEngine())
-var deployableByOlmCheck certification.Check = &operatorpol.DeployableByOlmCheck{}
 var operatorPkgNameIsUniqueCheck certification.Check = &operatorpol.OperatorPkgNameIsUniqueCheck{}
-var validateOperatorBundle certification.Check = operatorpol.NewValidateOperatorBundleCheck(internal.NewOperatorSdkEngine())
 var scorecardBasicSpecCheck certification.Check = operatorpol.NewScorecardBasicSpecCheck(internal.NewOperatorSdkEngine())
```

Comment:
It would be great to have two blocks separated by a new line: one for operator checks, and one for container checks.

Reply:
That would make things clear for sure, if we do that we should comment which is which so no one comes along and adds in a new check in that empty line.

Reply:
Will do.